etc/docker/kubeflow/Dockerfile.dev

```diff
@@ -0,0 +1,72 @@
+# syntax=docker/dockerfile:experimental
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Ubuntu 20.04.2 LTS (Focal Fossa)
+# Repository: https://gallery.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter
+FROM public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter:v1.4
+
+ARG TAG="dev"
```

Comment:
NIT: the specified arg is not used and should be removed

Reply:
Fixed in https://github.com/elyra-ai/elyra/pull/2344/commits/592afbb1955024f2f3d32d8fcb9423b08c343d28